src/parsers/sample-models.hpp

```diff
@@ -20,6 +20,7 @@
 #define __se3_sample_models_hpp__
```

Comment:
The two new models are never sampled, but there is the name sample in the title which may lead to some confusions. Can you change it for something like {basic, simple} models?